Intact.BuinessLogic/Services/RoomsService.cs

```diff
@@ -58,33 +59,35 @@ public async Task<bool> GetAvailabilityAsync(int id, CancellationToken cancellat
 
     public async Task<bool> JoinAsync(Guid userId, int id, CancellationToken cancellationToken)
     {
+        var profileId = await profileAccessor.GetProfileIdAsync();
         if (await GetAvailabilityAsync(id, cancellationToken) &&
```

Comment:
I think we need to add a check for profiles of the same user and disallow them

Reply:
so in general this is for prohibiting to play for the same user with himself using different profiles (to raise wins/ratings), but we could do this by some general workflow (which would be useful to fix other potential issues), like denial of consequent logins, or log out prev accessToken if new login is made, maybe even some future IP checks for cheat/clone prevention